### PR TITLE
fix: add priority scheduler e2e tests [DET-7106]

### PR DESCRIPTION
**Pull request by nrajanee**
## Description

feat: add priority scheduler e2e tests [DET-7106]

## Test Plan

Task was to add e2e tests. 

[DET-7106]: https://determinedai.atlassian.net/browse/DET-7106?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by hamidzr** [comment]
are the webui changes intentional?

**Comment by nrajanee** [comment]
> are the webui changes intentional?

No they're not. those got added because I forgot to do git submodule update --init to remove that. Will remove them

File: e2e_tests/tests/cluster/test_master_restart.py [hunk 1]
```diff
@@ -4,33 +4,22 @@
...
-    if not managed_cluster.reattach:
+def _sanity_check(managed_cluster_restarts: ManagedCluster) -> None:
+    if not managed_cluster_restarts.reattach:
         pytest.skip()
 
-    managed_cluster.ensure_agent_ok()
+    managed_cluster_restarts.ensure_agent_ok()
 
 
 @pytest.fixture
```
Comment (ioga) [inline e2e_tests/tests/cluster/test_master_restart.py:25]: `restartable_managed_cluster` in the next line here is using `managed_cluster` fixture
1.  it probably should use `managed_cluster_restarts` instead
2. `managed_cluster` fixture was removed, how come it still works

File: webui/react/src/components/PageHeader/PageHeader.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ const breadCrumbRender = (route: BreadCrumbRoute, params: unknown, routes: Bread
 };
 
 const PageHeader: React.FC<Props> = (props: Props) => {
-  const classes = [ css.base, props.className ];
+  const classes = [ css.base ];
```
Comment (ioga) [inline webui/react/src/components/PageHeader/PageHeader.tsx]: probably need to rebase or `git checkout master -- webui/` to get rid of these?

**Review by ioga — APPROVED** [review]

**Pull request merged**